src/NServiceBus.Core/Conventions.cs

```diff
@@ -34,15 +34,7 @@ internal IEnumerable<DataBusPropertyInfo> GetDataBusProperties(object message)
             return value;
         }
 
-        /// <summary>
-        ///     Returns the time to be received for a give <paramref name="messageType" />.
-        /// </summary>
-        public TimeSpan GetTimeToBeReceived(Type messageType)
```

Comment:
should be here: https://github.com/Particular/NServiceBus/pull/2737/files#diff-0837a2651d7b11c523bd240eefc155b0R487